assignments/array-methods.js

```diff
@@ -56,27 +56,101 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
 let fullName = [];
+runners.forEach((runner) => {
+    fullName.push(`${runner.first_name} ${runner.last_name}`)
+});
+
 console.log(fullName);
 
```

Comment:
love that you're passing in runner instead of a standard placeholder like element or item. reading this code is fun and easy.